public/index.html

```diff
@@ -0,0 +1,741 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brand Marketing App</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Project Week One">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300,100,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Condensed:400,700,300' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,100,300,500,600' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' href='css/stylesheet.css' type='text/css'>
+  </head>
+  <body>
+    <section class="home" id="home">
+      <section class="a1-home-header">
+        <section class="appbox-logo-section">
+          <img class= "appbox-logo" src="./img/appbox.png" alt="appbox logo">
+        </section>
+        <nav class="home-nav">
```

Comment:
This isn't really the navigation for `home`, though - it's the navigation for the whole of the page (or the site).

This, and the logo, should be in their own section outside of `#home.home`, since they're a self-contained concept - the site navigation - despite the fact that the background styling is shared between the top section (`#home`) and the navigation. Look to other sites (GitHub or Google, for example) for how they've solved this problem.